packages/app/package.json

```diff
@@ -63,6 +63,7 @@
     "ky": "^0.30.0",
     "ky-universal": "^0.10.1",
     "lodash": "^4.17.21",
+    "ms": "^2.1.3",
```

Comment:
Don't really need this package, but we have it on the backend and it seems small.. happy to remove it